### PR TITLE
1418 investigate icbadge cypress visual and a11y tests ci build failure

### PR DESCRIPTION
**Pull request by gd2910**
## Summary of the changes
Refactor of IcBadge cypress tests
## Related issue
#1418 
## Checklist
- [x] Relevant unit tests and visual regression tests added. 
- [x] Visual testing against Figma component specification completed. 
- [x] All acceptance criteria reviewed and met. 
- [x] Accessibility Insights FastPass performed.
- [x] A11y unit test added and yields no issues.
- [x] A11y plug-in on Storybook yields no issues. 
- [x] Manual screen reader testing performed using NVDA and VoiceOver. 
- [x] Page can be zoomed to 400% with no loss of content. 
- [x] Screen magnifier used with no issues. 
- [x] Text resized to 200% with no loss of content.
- [x] Text spacing increased as per the [WCAG 1.4.12 success criterion](https://www.w3.org/TR/WCAG21/#text-spacing) with no loss of content.
- [x] Browser setting 'prefers reduced motion' tested. No animations or motion visible whilst this setting is on. 
- [x] Windows High Contrast mode tested with no loss of content.
- [x] System light and dark mode tested with no loss of content. 
- [x] Manual keyboard testing for keyboard controls and logical focus order. 
- [x] Min/max content examples tested with no loss of content or overflow. 
- [x] Browser support tested (Chrome, Safari, Firefox and Edge).
- [x] Correct roles used and ARIA attributes used correctly where required. 
- [x] Logical heading structure is maintained, and the HTML elements used for headings can be changed to fit within the wider page structure. 
- [x] All prop combinations work without issue. 
- [x] Changes to docs package checked and committed.

### CODE REVIEW

File: packages/react/src/component-tests/IcToast/IcToast.cy.tsx [hunk 1]
```diff
@@ -125,31 +124,31 @@ describe("IcToast Visual Regression and A11y Testing", () => {
...
+    cy.wait(150).checkA11y;
   });
 
   it("renders slotted link", () => {
     mount(<SlottedLinkToast />);
     cy.get("ic-button").click();
 
     cy.wait(100).compareSnapshot("slotted-link", DEFAULT_TEST_THRESHOLD + 0.02);
-    cy.wait(100).checkA11y(undefined, CYPRESS_AXE_OPTIONS);
+    cy.wait(100).checkA11y;
```
Comment (GCHQ-Developer-299) [inline packages/react/src/component-tests/IcToast/IcToast.cy.tsx:135]: What makes cy.wait(100).checka11y different to cy.checkA11yWithWait(undefined, 100) ?

File: packages/react/cypress/utils/a11y.ts [hunk 2]
```diff
@@ -12,3 +12,20 @@ export const CYPRESS_AXE_OPTIONS = {
     region: { enabled: false },
   },
 };
+
+export const CYPRESS_AXE_REPORTING = (axeResults) => {
```
Comment (GCHQ-Developer-299) [inline packages/react/cypress/utils/a11y.ts:16]: Love this, super useful :)

**Review by GCHQ-Developer-299 — DISMISSED** [review]

File: packages/docs/CHANGELOG.md [hunk 3]
```diff
@@ -2,3 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 2.9.1 (2024-01-25)
+
+# 2.9.0 (2024-01-15)
```
Comment (GCHQ-Developer-465) [inline packages/docs/CHANGELOG.md:8]: Does this need removing?

File: packages/web-components/CHANGELOG.md [hunk 4]
```diff
@@ -2,3 +2,17 @@
...
+
+- **web-components:** updated accessible label for nav sections in page-header and top-nav ([eb74238](https://github.com/mi6/ic-ui-kit/commit/eb74238da159f0f3b4a09bb63cf4aef8a0855d4c))
+- **web-components:** updated manualInputKeyboardOpen function ([92c2f89](https://github.com/mi6/ic-ui-kit/commit/92c2f89ea1e7ba33d19bab54c21b4ce6de56bbf5))
+
+### Features
+
+- **web-components:** add dropdown option to buttons ([6660857](https://github.com/mi6/ic-ui-kit/commit/6660857cf788fd832bb05ae1aadf2f8903bb5b6b)), closes [#1321](https://github.com/mi6/ic-ui-kit/issues/1321)
+- **web-components:** updated web-components to include props required for canary components ([a957b05](https://github.com/mi6/ic-ui-kit/commit/a957b056335e3d0cd3129fa24c91e30ec9ca0780))
+
+# 2.9.0 (2024-01-15)
```
Comment (GCHQ-Developer-465) [inline packages/web-components/CHANGELOG.md:18]: Does this need removing?

**Review by GCHQ-Developer-465 — COMMENTED** [review]
Looks good 👍 Just a small comment.

**Review by GCHQ-Developer-465 — APPROVED** [review]

**Review by GCHQ-Developer-299 — APPROVED** [review]

**Pull request merged**